src/general/PauseMenu.tscn

```diff
@@ -291,7 +291,7 @@ anims/Close = SubResource( 1 )
 anims/Open = SubResource( 2 )
 anims/RESET = SubResource( 3 )
 
-[connection signal="pressed" from="CenterContainer/PrimaryMenu/Resume" to="." method="ClosePressed"]
+[connection signal="pressed" from="CenterContainer/PrimaryMenu/Resume" to="." method="ResumePressed"]
```

Comment:
These are the kind of painful changes for me that I need to all redo for my later prototypes...

Also the Closed signal being renamed to ClosePressed(). If there's no really good reason to change them, could we not change them, at least for a month until my prototype is merged? At which point all the scenes would be in master that need modifying at once.

Reply:
Sure.